test/integration/smoke-testing/test-my-stuff.js

```diff
@@ -26,8 +26,6 @@ var password = process.env.SMOKE_PASSWORD;
 var rootUrl = process.env.ROOT_URL || 'https://scratch.ly';
 var url = rootUrl + '/users/' + username;
 
-tap.plan(7);
```

Comment:
Is removing the plan necessary?

Reply:
It probably isn't.  It seems like we should look into how necessary tap.plan is in a separate pr.  I'll update this one.